Enrise/epic-cs
# EF-179 creation of a code style repository in order to maintain standards between our projects

### What

epic-cs has been created as a command line tool to pull in to a project our latest code styles.

It will move files and create a STYLE.md file containing instructions and setup scripts to be run.

### How to test

1. Follow the instructions from the readme for installation
1. Navigate to your project directory (where you want to update or add the code style)
1. Run epic-cs and verify it correctly copies the selected setup files and commands/instructions

Consider:
- Is this useful?
- Does the automation serve any purpose? Or could it just be a repo containing static files and instructions?
- Or should it be automated further to actually install everything? How far should it go?
